dablin.spec

```diff
@@ -31,6 +31,7 @@ Summary:  DAB/DAB+ receiver for Linux (including ETI-NI playback)
 License:  GPLv3+ and LGPLv2.1+
 URL:      https://github.com/Opendigitalradio/dablin
 Source0:  https://github.com/Opendigitalradio/dablin/archive/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Patch0:   https://github.com/Opendigitalradio/dablin/compare/master...radiorabe:feature/fix-build-on-legacy-gtkmm.patch
```

Comment:
Maybe it would be better to compare against the 1.4.1 tag instead of the master to ensure a stable patch URL (in case the master changes):
`Patch0:   https://github.com/Opendigitalradio/dablin/compare/%{version}...radiorabe:feature/fix-build-on-legacy-gtkmm.patch`